[2024-11-03T16:16:38.982+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-03T16:16:38.989+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.get_metadata manual__2024-11-03T16:16:37.934572+00:00 [queued]>
[2024-11-03T16:16:38.992+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.get_metadata manual__2024-11-03T16:16:37.934572+00:00 [queued]>
[2024-11-03T16:16:38.993+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-03T16:16:38.999+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): get_metadata> on 2024-11-03 16:16:37.934572+00:00
[2024-11-03T16:16:39.001+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=133) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-03T16:16:39.002+0000] {standard_task_runner.py:72} INFO - Started process 137 to run task
[2024-11-03T16:16:39.003+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'get_metadata', 'manual__2024-11-03T16:16:37.934572+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpej1udenk']
[2024-11-03T16:16:39.004+0000] {standard_task_runner.py:105} INFO - Job 65: Subtask get_metadata
[2024-11-03T16:16:39.022+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.get_metadata manual__2024-11-03T16:16:37.934572+00:00 [running]> on host 2b8c286c59ee
[2024-11-03T16:16:39.059+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='get_metadata' AIRFLOW_CTX_EXECUTION_DATE='2024-11-03T16:16:37.934572+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-03T16:16:37.934572+00:00'
[2024-11-03T16:16:39.059+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-03T16:16:40.700+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-03T16:16:40.708+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-03T16:16:40.708+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=get_metadata, run_id=manual__2024-11-03T16:16:37.934572+00:00, execution_date=20241103T161637, start_date=20241103T161638, end_date=20241103T161640
[2024-11-03T16:16:40.759+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-03T16:16:40.773+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-03T16:16:40.774+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
